src/views/Dashboard.vue

```diff
@@ -203,6 +206,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+	@import '../assets/buttons';
```

Comment:
this is not necessary anymore. The button component takes care of all this.

```suggestion
```

Reply:
Ok